### PR TITLE
[CUBRIDMAN-104] Modify the specifications for column property on ALTER TABLE clause

### PR DESCRIPTION
**Pull request by beyondykk9**
http://jira.cubrid.org/browse/CUBRIDMAN-104

### CODE REVIEW

**Review by ctshim — APPROVED** [review]

File: ko/sql/schema/table_stmt.rst [hunk 1]
```diff
@@ -1614,15 +1614,21 @@ CHANGE/MODIFY 절
 
 *   **NOT NULL**
 
-    *   변경할 칼럼에 **NOT NULL** 제약 조건이 지정되지 않으면 기존 테이블에 존재하더라도 새 테이블에서 제거된다.
-    *   변경할 칼럼에 **NOT NULL** 제약 조건이 지정되면 시스템 파라미터 **alter_table_change_type_strict** 의 설정에 따라 결과가 달라진다.
+    *   변경할 칼럼에 **NOT NULL** 속성이 지정되지 않더라도 새 테이블에서 해당 속성이 제거되지 않고 그대로 남아있게 된다. 새 테이블에서 해당 속성을 제거하려면 구문에서 ** NULL **\을 지정하면 된다.
```
Comment (youngjinj) [inline ko/sql/schema/table_stmt.rst]: Please remove spaces before and after "NULL".
```
** NULL **\을
```
to
```
**NULL**\을
```

Comment (youngjinj) [inline ko/sql/schema/table_stmt.rst]: Shouldn't "constraint" be translated as "제약조건"? The term "제약조건" is more natural than "속성".
If necessary, the term "속성" in all other locations should be replaced with "제약조건".

File: ko/sql/schema/table_stmt.rst [hunk 2]
```diff
@@ -1614,15 +1614,21 @@ CHANGE/MODIFY 절
...
+    *   변경할 칼럼에 **NOT NULL** 속성이 지정되면 시스템 파라미터 **alter_table_change_type_strict** 의 설정에 따라 결과가 달라진다.
 
         *   **alter_table_change_type_strict** 가 yes이면 해당 칼럼의 값을 검사하여 **NULL** 이 존재하면 오류가 발생하고 변경을 수행하지 않는다.
         *   **alter_table_change_type_strict** 가 no이면 존재하는 모든 **NULL** 값을 변경할 타입의 고정 기본값(hard default value)으로 변경한다.
 
-*   **DEFAULT**: 변경할 칼럼에 **DEFAULT** 속성이 지정되지 않으면 이 속성이 기존 테이블에 있더라도 새 테이블에서 제거된다.
+*   **DEFAULT**: 변경할 칼럼에 **DEFAULT** 속성이 지정되지 않더라도 새 테이블에서 해당 속성이 제거되지 않고 그대로 남아있게 된다. 새 테이블에서 해당 속성을 제거하려면 구문에서 ** DEFAULT NULL **\을 지정하면 된다.
```
Comment (youngjinj) [inline ko/sql/schema/table_stmt.rst]: Please remove spaces before and after "DEFAULT NULL".
```
** DEFAULT NULL **\을
```
to
```
**DEFAULT NULL**\을
```

File: ko/sql/schema/table_stmt.rst (hunk 1 above)
Comment (beyondykk9) [reply ko/sql/schema/table_stmt.rst]: Thanks, I will fix it, also engligh version.

File: ko/sql/schema/table_stmt.rst (hunk 2 above)
Comment (beyondykk9) [reply ko/sql/schema/table_stmt.rst]: I fixed it, thanks.

File: ko/sql/schema/table_stmt.rst (hunk 1 above)
Comment (beyondykk9) [reply ko/sql/schema/table_stmt.rst]: I fixed it, thanks.

**Review by hyunikn — APPROVED** [review]

**Review by shparkcubrid — APPROVED** [review]

**Review by youngjinj — APPROVED** [review]

File: en/sql/schema/table_stmt.rst [hunk 3]
```diff
@@ -1615,15 +1615,21 @@ Changes of Table Attributes based on Changes of Column Type
...
+*   **DEFAULT** : Even though the **DEFAULT** attribute is not specified in the column to change, it will not be removed from a new table if the attribute is present in the existing table. If you want to remove the attribute **DEFAULT**, specify the **DEFAULT NULL** in the statement.
 
-*   **AUTO_INCREMENT** : If the **AUTO_INCREMENT** attribute is not specified in the column to change, it will be removed from a new table even though the attribute is present in the existing table.
+*   **COMMENT** : Even though the **COMMENT** attribute is not specified in the column to change, it will not be removed from a new table if the attribute is present in the existing table. If you want to remove the attribute **COMMENT**, specify the **COMMENT ''** in the statement.
+
+*   **AUTO_INCREMENT** : Even though the **AUTO_INCREMENT** attribute is not specified in the column to change, it will not be removed from a new table if the attribute is present in the existing table.
+        *   caution) The **AUTO_INCREMENT** attribute can not removed from a new table once the attribute is set by CREATE or ALTER.
```
Comment (charischau) [inline en/sql/schema/table_stmt.rst]:
```suggestion
        *   caution) The **AUTO_INCREMENT** attribute can not be removed from a new table once the attribute is set by CREATE or ALTER.
```

File: en/sql/schema/table_stmt.rst [hunk 4]
```diff
@@ -1615,15 +1615,21 @@ Changes of Table Attributes based on Changes of Column Type
...
+
+*   **ON UPDATE** : Even though the **ON UPDATE** attribute is not specified in the column to change, it will not be removed from a new table if the attribute is present in the existing table.
+        *   caution) The **ON UPDATE** attribute can not removed from a new table once the attribute is set by CREATE or ALTER.
```
Comment (charischau) [inline en/sql/schema/table_stmt.rst]:
```suggestion
        *   caution) The **ON UPDATE** attribute can not be removed from a new table once the attribute is set by CREATE or ALTER.
```

**Review by charischau — APPROVED** [review]

**Review by hgryoo — APPROVED** [review]

**Pull request merged**